src/components/Posts/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+import { Hearts } from 'react-loader-spinner';
+
+import { getPosts } from 'api/post.service';
+
+import Post from 'components/Post';
+import Box from '../Box';
+
+import './style.scss';
+
+const Posts = () => {
+  const [isFetchingPosts, setIsFetchingPosts] = useState(true);
+  const [posts, setPosts] = useState([]);
+
+  useEffect(() => {
+    getPosts()
+      .then((response) => {
+        setPosts(response);
+      })
+      .catch(() => {
+        toast.error('There was an error. Please try again later.');
+      })
+      .finally(() => {
+        setIsFetchingPosts(false);
+      });
+  }, []);
+
+  return (
+    <div className="posts__container">
```

Comment:
this is the global container so it would be the BLOCK of BEM
```suggestion
    <div className="posts">
```